doc/ws.md

```diff
@@ -71,8 +71,8 @@ provided then that is extension parameters:
   takeover.
 - `serverMaxWindowBits` {Number} The value of `windowBits`.
 - `clientMaxWindowBits` {Number} Request a custom client window size.
-- `level` {Number} The value of zlib's `level` param (0-9, default 8).
-- `memLevel` {Number} The value of zlib's `memLevel` param (1-9, default 8).
+- `zlibOptions` {Object} [Additional options][zlib-options] to pass to zlib
```

Comment:
`zlibDeflateOptions`, we also need to add `zlibInflateOptions`, I can do after merge if it's too boring.